springfox-swagger-common/build.gradle

```diff
@@ -16,18 +16,25 @@ dependencies {
   compile project(':springfox-spring-web')
   compile libs.core
   compile libs.spring
+  compile group: 'org.springframework.boot', name: 'spring-boot-autoconfigure', version: springBoot
 
   provided libs.springProvided
   provided libs.clientProvided
   provided libs.springProvidedWithMvc
+  provided group: 'io.projectreactor', name: 'reactor-core', version: '3.2.6.RELEASE'
```

Comment:
Why are we using project reactor?

Reply:
As the `CsrfTokenWebFluxLoader` did use it a little bit (the `Mono`), but I'll reorganize them.